bundle/src/bootstraps/standalone.commercial.ts

```diff
@@ -221,7 +221,7 @@ const chooseAdvertisingTag = async () => {
 	// Only load the Opt Out tag in TCF regions when there is no consent for Googletag
 	if (consentState.tcfv2 && !getConsentFor('googletag', consentState)) {
 		// Don't load OptOut (for now) if loading Elements Manager
-		if (!isInVariantSynchronous(elementsManager, 'variant')) {
+		if (isInVariantSynchronous(elementsManager, 'variant')) {
```

Comment:
Will this also skip loading the regular commercial bundle if the user is in the elements manager test?

Reply:
Ah yes :)